src/themes/move-theme/typography.js

```diff
@@ -5,12 +5,9 @@ This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
 */
 // @flow
-import type {FontTokensT} from '../types.js';
 
-export const fontTokens: FontTokensT = {
```

Comment:
I'd keep it as `...tokens` or anything extendable, so we can potentially include the secondary font in it.